scripts/v.db.update/v.db.update.html

```diff
@@ -103,8 +103,3 @@ <h2>SEE ALSO</h2>
 <h2>AUTHOR</h2>
 
 Moritz Lennert (mlennert@club.worldonline.be)
-
-<!--
-<p>
-<i>Last changed: $Date$</i>
--->
```

Comment:
Line 7, 14, 88, 102 unnecessary one empty line.

Line 98

`<a href="v.db.select.html">v.db.select</a><br>`

should be 

`<a href="v.db.select.html">v.db.select</a>,`.